pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "lsst-ts-observing"
+name = "ts-observing"
```

Comment:
Okay, I'm concerned with this naming convention since it seems quite generic. Are telescope and site creating lots of conda packages without the lsst prefix? DM has gone the opposite route and made clear that all our PyPI packages use `lsst-` prefix. Did we all decide on conda/pypi naming conventions at some point and DM ignored something? cc/ @ktlim

Reply:
So, tssw do not use pypi for any package, all our packages uses conda. In conda, we host all our packages in the `lsst-ts` organization rather than in `condo-forge`, mostly because we want to have finer control of the versions we make available. We also build the conda packages in our Jenkins instance. In any case, we do not use the `lsst-` prefix in any of our packages.

Reply:
Okay, we need to remember to have a whole new discussion if you do end up wanting to put something on PyPI or conda-forge 😄